agenta-web/src/pages/apps/[app_id]/annotations/human_a_b_testing/[evaluation_id]/index.tsx

```diff
@@ -1,5 +1,5 @@
 import ABTestingEvaluationTable from "@/components/EvaluationTable/ABTestingEvaluationTable"
-import {Evaluation} from "@/lib/Types"
+import type {Evaluation} from "@/lib/Types"
```

Comment:
@bekossy do we need this. It looks a bit strange here?

Reply:
I'm getting an error message whenever I open that page, the error message: `Import 'Evaluation' conflicts with local value, so must be declared with a type-only import when 'isolatedModules' is enabled.` and the build also sometimes gets failed for that.

Reply:
yeah it's a type import nothing to worry about